extensions/panache/panache-hibernate-common/runtime/src/main/java/io/quarkus/panache/hibernate/common/runtime/PanacheJpaUtil.java

```diff
@@ -191,4 +195,30 @@ public static String toOrderBy(Sort sort) {
         }
         return sb.toString();
     }
+
+    private static StringBuilder escapeColumnName(String columnName) {
+        StringBuilder sb = new StringBuilder();
+        String[] path = columnName.split("\\.");
```

Comment:
Can we have dots in property names? I assume not. I'm asking because if there's a way to escape dots like `\.` in column names, then we'd have to respect that, but I don't think column names support dot escaping.

I just cheked the HQL grammar and there are no escaped dots, so we're good.